spec/services/publishers/vacancies/vacancy_step_process_spec.rb

```diff
@@ -135,7 +129,7 @@
 
       context "when the vacancy is at multiple schools" do
         let(:school) { create(:school, :secondary) }
-        let(:vacancy) { create(:vacancy, organisations: [school, school2]) }
+        let(:vacancy) { create(:vacancy, phases: %w[primary], organisations: [school, school2]) }
 
         context "when the schools have the same phase" do
           let(:school2) { create(:school, :secondary) }
```

Comment:
Seems this test is no longer relevant? 

As far as I understand, we now only show the education phases step if the vacancy has no school phases. In this test, both schools have a phase.